extras/group-virt.example

```diff
@@ -22,3 +22,4 @@ server.tcp-user-timeout=20
 server.keepalive-time=10
 server.keepalive-interval=2
 server.keepalive-count=5
+cluster.lookup-optimize=off
```

Comment:
nit: should we sort the options alphabetically, so it'll appear around line 13 or so?

Reply:
The existing options are not sorted. I'm not sure if it would provide any benefit. There aren't many...